lint-staged.config.js

```diff
@@ -0,0 +1,15 @@
+const rootDir = process.cwd();
```

Comment:
Are you sure that this will work correctly if the precommit hook is launched from a deeply nested project directory? I think we might want `require.main.filename`: 

```
const { dirname } = require('path');
const rootDir = dirname(require.main.filename);
```

If this works as-is, feel free to ignore

Reply:
Hmmm, yeah this way feels safer, will update